packages/joint-core/types/joint.d.ts

```diff
@@ -3486,7 +3486,7 @@ export namespace mvc {
 
         /**
          * Events hash or a method returning the events hash that maps events/selectors to methods on your View.
-         * For assigning events as object hash, do it like this: this.events = <any>{ "event:selector": callback, ... };
+         * For assigning events as object hash, do it like this: `this.events = <any>{ "event:selector": callback, ... };`
```

Comment:
why is this here?

Reply:
This JSDoc was relevant for me when doing the conversion to TS, and the missing `` broke the formatting of the code example so that it did not make sense, so I fixed it.

I can remove it from this PR if you want